tests/cli/keyboard_interface.c

```diff
@@ -59,63 +53,61 @@ int connect_tcp() {
     servaddr.sin_port = htons(PORT);
 
     if ((bind(sockfd, (struct sockaddr*) &servaddr, sizeof(servaddr))) != 0) {
-        printf("socket bind failed... %s\n", strerror(errno));
+        printf("Keyboard interface: socket bind failed... %s\n", strerror(errno));
         exit(0);
     }
 
-    printf("Waiting for client\n");
+    printf("Keyboard interface: Keyboard waiting for client\n");
+    sleep(2);
```

Comment:
Is the `sleep(2)` necessary?

Reply:
This was just so that the print statements from the keyboard interface dont mix around with the cli's print statements which is what it was doing w/o the sleep

Reply:
Can you move the `sleep()` to the CLI instead? Since this sleep is more of an "aesthetic" thing, it's more appropriate for the CLI